docs/Known_SetUp_Issues.adoc

```diff
@@ -0,0 +1,162 @@
+== Known Set Up Issues on a Mac [[issues-on-mac]]
+
+In this document we will explain how one could setup their Mac for AgnosticD deployment with OpenStack and the known issues to date.
+
+If you are a Red Hatter, you can use link:labs.opentlc.com[labs.opentlc.com] to request access to our OpenStack cluster.
+. Log in to link:https://labs.opentlc.com[https://labs.opentlc.com] using your opentlc user.
+. Go to *Services* -> *Catalogs* -> *Novello testing* -> *DEV OSP Sandbox*.
+. Click *Order* -> *Submit*
+
+1. Error when executing test playbook:
++
+[source, shell]
+----
+sassenach:ansible Cibeles$ ansible-playbook main.yml \ 
+    -e @configs/test-empty-config/sample_vars_osp.yml \ 
+    -e @~/.config/openstack/secret.yml
+[...]
+TASK [infra-osp-dry-run : Delete Heat stack that doesn't exist] ********************************************************
+Monday 25 May 2020  15:48:17 +0200 (0:00:00.832)       0:00:11.583 ************
+fatal: [localhost]: FAILED! => {"changed": false, "msg": "openstacksdk is required for this module"}
+	to retry, use: --limit @/Users/Cibeles/repos/agnosticd/ansible/main.retry
+
+PLAY RECAP *************************************************************************************************************
+localhost                  : ok=16   changed=2    unreachable=0    failed=1
+----
+
+You probably forgot to create or activate your virtual env, to fix this simply run:
++
+[source, shell]
+----
+$ source /Users/[USER]/.pyenv/versions/openstack-ansible-2.9/bin/activate
```

Comment:
Pyenv? Why?

Reply:
it's on the OSP conf steps, without that, I could not get it working